### PR TITLE
AWS Quick Starts with the Pulumi Registry

### PR DESCRIPTION
**Pull request by ajhool**
Notes:

  - I'm not sure whether the "AWS Quick Start" name should be used when referring to the components or whether more phrasing should be "based on AWS Quick Starts"
  - I did not add a meta image. It looks like these are usually custom made

### CODE REVIEW

**Comment by susanev** [comment]
@mattstratton it looks like ur suggestions have been addressed, did ya wanna review again?

**Review by mattstratton — APPROVED** [review]
This looks great! I am good to merge. Sorry for the delay!

**Comment by susanev** [comment]
@ajhool would you mind adding in this meta img and updating the blog post date? then we can get this merged.

meta

**Comment by ajhool** [comment]
@susanev - Done, thanks! Having fresh eyes after some time, I think that the blog post is improved by adding a brief conclusion section that addresses the three problems discussed in the intro; so I've added that as well.

**Comment by susanev** [comment]
great, thank you!!

**Comment by cnunciato** [comment]
Hi @ajhool! 👋  We need a couple more little things before we can merge this:

- A TOML file containing your author info at `themes/default/data/team/team/aidan-hoolachan.toml`. You can use the following content for it (feel free to adjust, add your social info, etc., as you lke -- see others in that directory for examples):
    ```
    id = "aidan-hoolachan"
    name = "Aidan Hoolachan"
    status = "guest"
    ```
- An image to accompany your author record at `themes/default/static/images/team/aidan-hoolachan.jpg` This must be a JPG and is generally 400x400, but can be a bit smaller. Should be square, though.

Thanks! Once we have these, we should be able to merge.

**Comment by susanev** [comment]
@ajhool sry about the back and forth on this. are you able to make the updates @cnunciato listed? or let us know if you need any help in any way.

would absolutely love to get your blog published!

**Comment by ajhool** [comment]
@susanev yes, I will do this ASAP, thanks!

**Comment by ajhool** [comment]
@susanev added author information :)

File: themes/default/content/blog/aws-quickstart-registry/index.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
+---
+title: "Using AWS Quick Starts with the Pulumi Registry"
+date: 2021-12-17
```
Comment (cnunciato) [inline themes/default/content/blog/aws-quickstart-registry/index.md:3]: Should we update this to today-ish?

```suggestion
date: 2022-01-05
```

**Review by cnunciato — APPROVED** [review]
Looks great, thanks @ajhool! One suggestion about the date, otherwise I think we can merge.

**Pull request merged**